Headers/DebugServer2/Architecture/X86/CPUState.h

```diff
@@ -33,17 +33,22 @@ struct AVXVector {
 };
 
 struct X87Register {
-  union {
-#if !defined(OS_WIN32)
-    // On Windows, long double and double are the same type.
-    // https://msdn.microsoft.com/en-us/library/9cx8xs15.aspx
-    long double f80;
-#endif
-    double f64;
-    float f32;
-    uint64_t mm;
-    uint8_t bytes[10];
-  };
+  uint8_t data[10];
+  uint8_t padding[6];
```

Comment:
Padding exposes implementation details. Remove it and create a constant with the amount of padding that can be used instead.